src/components/task.cjsx

```diff
@@ -105,7 +105,7 @@ module.exports = React.createClass
           {breadcrumbs}
           <BS.Panel bsStyle="default" footer={footer}>
             <h1>{model.title}</h1>
-            <p>Due At: {model.due_at}</p>
+            <p>Due At: {moment(model.due_at).format('LLL')}</p>
```

Comment:
Should we have a `<Time>` component that generates a `<time>` HTML element? That way date/time formatting is done consistently throughout the app?